Makefile

```diff
@@ -1,11 +1,5 @@
 check_dirs := score_analysis tests
```

Comment:
I don't think this line is needed any more.

Reply:
Indeed, no longer needed. Deleted it.